tests/rules/require-v-for-key.test.ts

```diff
@@ -139,6 +139,56 @@ div
           endColumn: 13
         }
       ]
+    },
+    {
+      filename: 'test.vue',
+      code: `<template lang="pug">
+.component-test(v-on="$listeners")
+  .bar(
+    v-for="(value, index) in values"
+    :style="getStyle(value, index)"
+  )
+  slot
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.component({
+  data() {
+    return {
+      values: [],
+    };
+  },
+  methods: {
+    getStyle(value, index) {
+      return \`opacity: 0.3\`
+    }
+  }
+});
+</script>
+
+<style lang="scss">
+.component-test {
+  height: 22px;
+
+  .bar {
+    width: 2px;
+    height: 100%;
+    margin-right: 2px;
+  }
+}
+</style>
+`,
```

Comment:
That part is not important for the test

```suggestion
</template>`,
```